app/views/project_submissions/flags/new.html.erb

```diff
@@ -2,9 +2,19 @@
   <%= turbo_frame_tag 'flag_form', class: 'w-full' do %>
     <%= form_with model: [@project_submission, @flag], builder: TailwindFormBuilder do |form| %>
       <div class=''>
-        <div class="space-y-6 pt-4">
-          <div>
-            <%= form.text_area :reason, placeholder: 'Please explain why you are reporting this submission...', rows: 6, data: { test_id: 'flag-description-field' }, autofocus: true %>
+        <div class="space-y-5 px-4">
```

Comment:
Just a few little design and markup suggestions for a final polish ✨ 

It'll be easier to just paste in the full final thing instead of leaving you lots of little minor comments:

```erb
<%= render ModalComponent.new(title: 'Report submission')  do %>
  <%= turbo_frame_tag 'flag_form', class: 'w-full' do %>
    <%= form_with model: [@project_submission, @flag], builder: TailwindFormBuilder do |form| %>

      <div class="space-y-5">
        <% Flag::REASONS.each do |reason| %>
          <div class="relative flex items-center">
            <div class="absolute flex h-6 items-center">
              <%= form.radio_button :reason, reason.name, data: { test_id: "flag-reason-#{reason.name}"}, class: 'h-4 w-4 border-gray-300 dark:border-gray-500 dark:bg-gray-700/50 dark:checked:bg-gold-600 dark:checked:border-gold-600 dark:focus:ring-offset-gray-800 text-gold-600 focus:ring-gold-600' %>
            </div>

            <div class="pl-7 text-sm leading-6">
              <%= form.label :reason, reason.description, value: reason.name, class: 'block text-sm font-medium text-gray-700 dark:text-gray-200 dark:text-gray-200' %>
            </div>
          </div>
        <% end %>
      </div>

      <div class="pt-5">
        <%= form.text_area :extra, placeholder: 'Please explain why you are reporting this submission...', rows: 6, data: { test_id: 'flag-description-field' }, autofocus: true %>
        <p class="mt-3 text-sm leading-6 text-gray-600 dark:text-gray-400"> Please include as much detail as possible to help us assess and action your report.</p>
      </div>

      <div class="mt-5 sm:flex sm:flex-row-reverse pt-4">
        <%= form.submit 'Save', class: 'cursor-pointer inline-flex w-full justify-center rounded-md button--primary px-5 py-2 text-sm font-semibold text-white shadow-sm sm:ml-3 sm:w-auto', data: { test_id: 'submit-btn' } %>
        <button type="button" data-action="click->modal#close" class="button--white mt-3 inline-flex w-full justify-center sm:mt-0 sm:w-auto">Cancel</button>
      </div>
    <% end %>
  <% end %>
<% end %>
```

Heres whats changed and why:
* We can wrap the radio button list in a containing div so the `space-x-5` doesn't affect the text area.
* Dropping the px-4 from the reasons list aligns it with the title and text area. 
* Few little adjustments to reasons - centering the label with the radio control and reducing the space between the label and control. 
* Separating out the different parts of the form with newlines between the markup makes it a little easier to parse/read.
* Giving the container for the textarea a `pb-5` instead of `mt-4`  to match the padding between the modal title and reason list creates a nice visual hierarchy.

The final thing will look this this:

<img width="514" alt="Screenshot 2023-09-02 at 12 15 35" src="https://github.com/TheOdinProject/theodinproject/assets/7963776/3604dcfa-e99a-4cc0-b3d5-0fd8772ded65">